hyprctl/main.cpp

```diff
@@ -51,73 +52,73 @@ const std::string USAGE = R"#(usage: hyprctl [(opt)flags] [command] [(opt)args]
 )#";
 
 void              request(std::string arg, int minArgs = 0) {
-    const auto SERVERSOCKET = socket(AF_UNIX, SOCK_STREAM, 0);
+                 const auto SERVERSOCKET = socket(AF_UNIX, SOCK_STREAM, 0);
```

Comment:
?

Reply:
Idk why, but `clang-format` is messing up stuff, I mentioned it in an above comment

Reply:
ftfy

Reply:
Thanks!